.eslintrc

```diff
@@ -2,5 +2,9 @@
   "extends": "plugin:shopify/esnext",
   "env": {
     "browser": true
+  },
+  "rules": {
+  	"promise/always-return": 0,
```

Comment:
we should be able to disable a particular rule on a line/block instead of disabling it entirely